addons/mail/static/src/components/activity/activity.xml

```diff
@@ -114,10 +120,10 @@
                     </t>
 
                     <t t-if="activityView.activity.canWrite">
-                        <div name="tools" class="o_Activity_tools">
+                        <div name="tools" class="o_Activity_tools d-flex">
```

Comment:
I think there is a problem with the split in different PR here, this and everything below should not be part of the `dl` changes.

Reply:
Confirmed, thanks